client/homescreen/activity-panel/stock/index.js

```diff
@@ -40,13 +41,24 @@ export class StockPanel extends Component {
 	}
 
 	async updateStock( product, quantity ) {
-		const { invalidateResolution, updateProductStock } = this.props;
+		const {
+			invalidateResolution,
+			updateProductStock,
+			products,
+		} = this.props;
 
 		const result = await updateProductStock( product, quantity );
 
 		if ( result.success ) {
 			// Request more low stock products.
 			invalidateResolution( 'getItems', [ 'products', productsQuery ] );
+			if ( products.length < 2 ) {
+				invalidateResolution( 'getItemsTotalCount', [
+					'products',
+					getLowStockCountQuery,
+					null,
+				] );
+			}
```

Comment:
Why is the Stock panel being modified in this PR? If there's an issue with the Stock panel, can you open a separate PR?

Reply:
Reverted the last commit and took out [the above changes](https://github.com/woocommerce/woocommerce-admin/pull/5706/commits/1a01312b54acbc0db9eab2fb4abb87fcbe469d96), these are no part of a different branch/PR -> https://github.com/woocommerce/woocommerce-admin/commit/16fb47b8b601c938ee58d32eb975f5b0a9265c45